frontend/src/app/shared/components/confirmation-window/confirmation-window.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="wrapper">
+    <h1 mat-dialog-title>{{title}}</h1>
+    <div mat-dialog-content [align]="'center'">
+        {{message}}
+    </div>
+    <div mat-dialog-actions [align]="'center'">
+        <button [ngClass]="cancelButtonOptions.class" *ngIf="isHasCancelButton" mat-flat-button
+                (click)="cancel()">{{cancelButtonOptions.label}}</button>
+        <button [ngClass]="acceptButtonOptions.class" *ngIf="isHasAcceptButton" mat-flat-button
+                (click)="accept()">{{acceptButtonOptions.label}}</button>
+    </div>
+</div>
```

Comment:
It would be better to have the ability to have as many buttons as the user wants